### PR TITLE
Introduced small stylesheets instead of one huge

### LINKED ISSUE #45: Create separated style files from style.scss.

### PR DESCRIPTION
**Pull request by postJScriptum**
Fixes: #45

Created separated small, partial style files from one big style.scss.

### CODE REVIEW

**Comment by pomek** [comment]
I tried to resolve a conflict but… I'm not sure whether I will do it properly. I don't want to remove some part of your job. Will be better if you resolve the conflict.

**Comment by pomek** [comment]
I noticed one thing for all created files. They don't have an empty line at the end.

[EditorConfig](http://editorconfig.org/) should care about it, but most probably you don't use it.

File: assets/styles/partials/_base.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	margin-bottom: 25px;
+}
+
+hr {
+	margin: 30px auto;
+}
+
+pre code {
+	white-space: inherit;
+    overflow: hidden;
```
Comment (pomek) [inline assets/styles/partials/_base.scss]: We use tab indent instead of spaces.

**Comment by postJScriptum** [comment]
Yes You had right, I didin't installed EditorConfig package to sublime-text. Now every scss file has empty line on the end.

**Pull request merged**